app/views/articles/_home_page.html.erb

```diff
@@ -5,8 +5,8 @@
     <span aria-hidden="true">&times;</span>
   </button>
   <h2>Welcome to articles+ beta!</h2>
-  <p>This is a pre-release version of the new article search feature, still in active development. Small updates will be happening frequently over the next 2 weeks while we finish up details.</p>
-  <p>Feel free to explore, and please send feedback!</p>
+    <p>This is an early release of the new article search feature, still in active development. Small updates will be happening frequently over the next 2 weeks while we finish up details.</p>
+    <p>Explore and let us know what you think! Your <a data-toggle="collapse" data-target="#feedback-form" href="#" class="collapsed" aria-expanded="false"> feedback</a> will help us improve the service.</p>
```

Comment:
Can we link this to `feedback_path` similar to the other [feedback link](https://github.com/sul-dlss/SearchWorks/blob/8ab334af06a76212f3257dcfcc0e0a3ca76e004d/app/views/shared/_top_navbar.html.erb#L19)?

This will ensure that the user gets to the form if the js doesn't trigger for whatever reason.

Reply:
Ah, good to know! Thanks @jkeck.